Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Pull base image.
+FROM ubuntu:14.04
+
+# Install Node.js
```

Comment:
you can probably just set the base image to use Node8 (alpine) or boron then you dont need to do the extra install steps for node8 which will save time on spinning up the container.

Reply:
https://hub.docker.com/_/node/ is a good place to see all the node images we can use. if you dont specify a distro in the image name i think its either ubuntu or debian.